ivy/functional/frontends/numpy/statistics/order_statistics.py

```diff
@@ -37,6 +48,82 @@ def _cpercentile(N, percent, key=lambda x: x):
     d0 = key(N[int(f)]) * (c - k)
     d1 = key(N[int(c)]) * (k - f)
     return d0 + d1
+import ivy
+
+
+@ivy.to_ivy_arrays_and_back
+@ivy.handle_numpy_out

```

Comment:
`@handle_numpy_out` should work. `ivy` does not have `handle_numpy_out` attribute